src/model/users.js

```diff
@@ -104,8 +105,23 @@ export const updateUser = async function (newUserData) {
   try {
     let password
     if (userToBeUpdated.password) {
+      // Compute passport new hash for local strategy
       password = await hashPassword(userToBeUpdated.password)
 
+      /* --- @deprecated : Update password hash for token strategy --- */
+      const passport = await PassportModelAPI.findOne({
+        protocol: 'token',
+        email: userToBeUpdated.email
+      })
+      if (passport) {
+        let shasum = crypto.createHash(passport.passwordAlgorithm)
```

Comment:
Are we sure that this passport.passwordAlgorithm will always have a value? I'm wondering if we shouldn't use this if there is a value and default to a Constant value which can be used in all the places where it needs to be used (instead of hardcoding sha512)

Reply:
It was used like this 
![image](https://user-images.githubusercontent.com/48720958/236437248-3b45ebe6-3963-4cff-9d99-9e66cd6207f7.png)
 
So I will updated to be the same

Reply:
done I addressed this with '||'